recipe/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+$PYTHON setup.py install --conda
+
+cp ${RECIPE_DIR}/xonsh_shortcut.json ${PREFIX}/Menu/xonsh_shortcut.json
+cp ${RECIPE_DIR}/xonsh.ico ${PREFIX}/Menu/xonsh.ico
```

Comment:
The `$PREFIX/Menu` directory does not exist on my system. I'm not sure that this is the right thing to do.  You may need to create such a directory first (`mkdir -p ${PREFIX}/Menu`).  Also, are you sure that it is `Menu` and not `menu`?